shared/common-adapters/oriented-image.desktop.js

```diff
@@ -51,43 +46,39 @@ class OrientedImage extends React.Component<Props, State> {
   }
 
   state = {
-    cacheStyleTransforms: {},
-    styleTransform: {transform: ''},
+    styleTransform: '',
   }
 
+  _cacheStyleTransforms = {}
```

Comment:
this actually should go up and be a module level cache, it can't exist in the insance